mvp-spec/partials-updater/src/interpretor.ts

```diff
@@ -137,6 +136,3 @@ function buildMermaidCommand(partialFile): { cmd: string; assetFile: string; } {
     return {cmd: cmd, assetFile: destFile};
 }
 
-function getPartialInCodeBlock(partial: string, lineBreak: string, type: string = "json"): string {
-    return `\`\`\`${type}${lineBreak}${partial}${lineBreak}\`\`\``;
-}
```

Comment:
I prefer with the extra function. It provide semantic to the cryptic line of code.

Reply:
Hmm... this cryptic line of code is just a concatenation :-D

Reply:
I've put back the function in a new commit.
I've used concatenation rather than template string to avoid the multiple \\`